### PR TITLE
Update use-group-policy.md

### PR DESCRIPTION
**Pull request by tseward**
Temporary fix for #1326.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by erikhoenderdos** [comment]
I am missing the folder id in this documentation. 

In the previous setup is was possible to auto sync a subfolder / teams channel. With this documentation you can only sync a complete teamsite. 

In example. If i want my customer to only sync the files of the General channel in a teamsite I needed to add the FOLDERID tag also. 

With the missing sync button as discussed in #1326 this was previously possible. Now it looks not possible anymore.

File: OneDrive/use-group-policy.md [hunk 1]
```diff
@@ -488,20 +488,24 @@ If you disable this setting, team site libraries that you've specified won't be
 
 To configure the setting, in the Options box, click **Show**, and then enter a friendly name to identify the library in the **Value Name** field and the entire library ID (tenantId=xxx&siteId=xxx&webId=xxx&listId=xxx&webUrl=httpsxxx&version=1) in the **Value** field. 
 
-To find the library ID, sign in as a global or SharePoint admin in Office 365, browse to the library, and click the **Sync** button. In the "Starting sync" dialog box, click the **Copy library ID** link.
+To find the tenant ID, install the [Azure Active Directory PowerShell for Graph](https://www.powershellgallery.com/packages/AzureAD).
 
-![The Getting ready to sync dialog box](media/copy-library-id.png)
+To find the site, web, and library IDs, install the [SharePoint PnP PowerShell module](https://www.powershellgallery.com/packages/SharePointPnPPowerShellOnline) and run the following PowerShell.
```
Comment (JohanFreelancer9) [inline OneDrive/use-group-policy.md]:
```suggestion
To find the site, web, and library IDs, install the [SharePoint PnP PowerShell module](https://www.powershellgallery.com/packages/SharePointPnPPowerShellOnline) and run the following PowerShell scripts.
```

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by kaarins** [comment]
I think this issue has been fixed. I updated the screenshot to show the new UI.

**Comment by dariomws** [comment]
Hi @kaarins, I think this issue is still not fixed. There are people who are still experiencing the same issue. Please check https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/1795.

Thanks!

**Pull request closed without merging**